boards/modalai/fc-v2/init/rc.board_sensors

```diff
@@ -24,4 +24,4 @@ icm42688p -s -b 2 -R 12 start
 bmm150 -I start
 
 # Internal I2C baro
-bmp388 -I start
+icp201xx -I start
```

Comment:
These boards have both?

Reply:
currently yes, but we are getting rid of the Bosch part in next board spins.  We left both on here in first build as a risk reduction if the TDK part didn't work.

We had the ICP20100 driver in our branch and we went to push it over Christmas and saw a couple days prior someone else pushed... doh!